briskedu/brisk-learning
# Fix Sentry error when document is blank

Summary:
Handling case where document is blank
https://app.asana.com/0/1203963408726580/1204220549201918
Test Plan:
1. Confirm on blank document flyout doesn't appear but there is no error
2. Confirm I can log in on blank document
